org.gnome.Evolution.json

```diff
@@ -0,0 +1,381 @@
+		"--share=network",
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+                "--talk-name=org.gnome.OnlineAccounts",
+                "--talk-name=org.freedesktop.Notifications",
+                "--talk-name=org.freedesktop.portal.Desktop"
+	],
+	"modules": [
+		{
+			"name": "libical",
+			"cleanup": [ "/lib/cmake" ],
+			"buildsystem": "cmake",
+			"config-opts": [
+				"-DCMAKE_INSTALL_LIBDIR:PATH=/app/lib",
+				"-DBUILD_SHARED_LIBS:BOOL=ON",
+				"-DGOBJECT_INTROSPECTION:BOOL=OFF"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/libical/libical/releases/download/v3.0.3/libical-3.0.3.tar.gz",
+					"sha256": "5b91eb8ad2d2dcada39d2f81d5e3ac15895823611dc7df91df39a35586f39241"
+				}
+			]
+		},
+
+		{
+			"name": "liboauth",
+			"config-opts": [ "--enable-nss" ],
+			"cleanup": [ "/share/man" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://downloads.sourceforge.net/project/liboauth/liboauth-1.0.3.tar.gz",
+					"sha256": "0df60157b052f0e774ade8a8bac59d6e8d4b464058cc55f9208d72e41156811f"
+				}
+			]
+		},
+
+		{
+			"name": "gnome-online-accounts",
+			"config-opts": [ "--disable-Werror", "--disable-telepathy", "--disable-documentation", "--disable-backend" ],
+			"cleanup": [ "/bin", "/share/GConf" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/gnome-online-accounts/3.28/gnome-online-accounts-3.28.0.tar.xz",
+					"sha256": "87bc4ef307604f1ce4f09f6e5c9996ef8d37ca5e0a3bf76f6b27d71844adb40c"
+				}
+			]
+		},
+
+		{
+			"name": "libgdata",
+			"config-opts": [
+				"--disable-always-build-tests",
+				"--disable-goa",
+				"--disable-gtk-doc",
+				"--disable-introspection",
+				"--disable-installed-tests",
+				"--disable-vala"
+			],
+			"cleanup": [
+				"/share/gtk-doc"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/libgdata/0.17/libgdata-0.17.9.tar.xz",
+					"sha256": "85c4f7674c0098ffaf060ae01b6b832cb277b3673d54ace3bdedaad6b127453a"
+				}
+			]
+		},
+
+		{
+			"name": "geocode-glib",
+			"config-opts": [
+				"--disable-introspection",
+				"--disable-gtk-doc"
+			],
+			"cleanup": [
+				"/share/gtk-doc"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/geocode-glib/3.24/geocode-glib-3.24.0.tar.xz",
+					"sha256": "19c1fef4fd89eb4bfe6decca45ac45a2eca9bb7933be560ce6c172194840c35e"
+				}
+			]
+		},
+
+		{
+			"name": "libgweather",
+			"config-opts": [
+				"--disable-glade-catalog",
+				"--disable-introspection",
+				"--disable-vala"
+			],
+			"cleanup": [
+				"/share/gtk-doc"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/libgweather/3.24/libgweather-3.24.1.tar.xz",
+					"sha256": "1227316037b63dac4153031a22abccb3d2ac679a7409e29cd303eec499eb653c"
+				}
+			]
+		},
+
+		{
+			"name": "openldap",
```

Comment:
this doesn't compile properly on aarch64. it needs to be reconfigured in a similar manner to this: https://github.com/flathub/shared-modules/blob/08e9b9eead7a95139be5d1fbc24bea730fb78f7a/SDL/SDL_net-1.2.8.json